### PR TITLE
Added comprehensive unit test suite for Mixin sink

### PR DESCRIPTION
**Pull request by TanmayRanaware**
- Created BDD-style test suite following existing patterns
- Tests cover plain objects, futures/promises, event listeners
- Includes edge cases and complex scenarios
- Validates sink configuration structure
- Ensures proper attribute application and removal

### CODE REVIEW

File: src/sinks/mixin-sink.test.ts [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+            expect(el.id).toBe('second-id');
+            expect(el.className).toBe('second-class');
+            expect(el.dataset.value).toBe('second');
+        });
+
+    });
+
+    describe('Given edge cases', () => {
+
+        it('handles empty mixin object', () => {
```
[the remark this answers is not in the document]
Comment (TanmayRanaware) [reply src/sinks/mixin-sink.test.ts]: right got you. Made change

File: QUICKSTART.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
+## Quick Start
```
[the remark this answers is not in the document]
Comment (TanmayRanaware) [reply QUICKSTART.md]: done

**Comment by dariomannu** [comment]
@TanmayRanaware:

8 tests failed:
✗ Mixin Sink > Given a plain object mixin > when applying attributes to element > when boolean attributes are provided > handles boolean attributes correctly [3.00ms]
✗ Mixin Sink > Given a plain object mixin > when applying attributes to element > when falsey values are provided > removes attributes when set to falsey values [2.00ms]
✗ Mixin Sink > Given a future/promise mixin > when applying future attributes > when promise resolves successfully > applies future attributes when promise resolves [18.00ms]
✗ Mixin Sink > Given event listener mixins > when handling future event listeners > when promise resolves with event handlers > handles future event listeners [14.00ms]
✗ Mixin Sink > Given complex mixin scenarios > when handling mixed attribute types in single mixin > when all attribute types are combined > handles mixed attribute types in single mixin [2.00ms]
✗ Mixin Sink > Given edge cases > when an empty mixin object is given > when no attributes are provided > handles empty mixin object
✗ Mixin Sink > Given edge cases > when null or undefined values are given > when attributes have null/undefined values > handles null and undefined values in mixin
✗ Mixin Sink > Given edge cases > when string "false" values are given > when attributes contain string "false" > handles string "false" values correctly [1.00ms]
